go.mod

```diff
@@ -119,44 +112,54 @@ require (
 	golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd
 	golang.org/x/tools v0.0.0-20200415034506-5d8e1897c761
 	gopkg.in/AlecAivazis/survey.v1 v1.8.3
-	gopkg.in/alecthomas/kingpin.v2 v2.2.6 // indirect
 	gopkg.in/src-d/go-git.v4 v4.13.1
 	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/api v0.0.0-20190816222004-e3a6b8045b0b
-	k8s.io/apiextensions-apiserver v0.0.0-20190718185103-d1ef975d28ce
-	k8s.io/apimachinery v0.0.0-20190816221834-a9f1d8a9c101
+	istio.io/api v0.0.0-20191115173247-e1a1952e5b81
+	istio.io/client-go v0.0.0-20191120150049-26c62a04cdbc
+	k8s.io/api v0.17.2
+	k8s.io/apiextensions-apiserver v0.0.0-20191114105449-027877536833
+	k8s.io/apimachinery v0.17.2
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
 	k8s.io/helm v2.7.2+incompatible
-	k8s.io/kube-openapi v0.0.0-20190228160746-b3a7cee44a30
-	k8s.io/kubernetes v1.11.3
+	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
+	k8s.io/kubernetes v1.14.0
 	k8s.io/metrics v0.0.0-20180620010437-b11cf31b380b
 	k8s.io/test-infra v0.0.0-20190131093439-a22cef183a8f
-	knative.dev/pkg v0.0.0-20191217184203-cf220a867b3d
+	knative.dev/pkg v0.0.0-20200207181514-32ea84581573
+	knative.dev/serving v0.12.1-0.20200210194206-365600fcbe27
 	sigs.k8s.io/yaml v1.1.0
 
 )
 
-replace k8s.io/api => k8s.io/api v0.0.0-20190528110122-9ad12a4af326
+replace k8s.io/api => k8s.io/api v0.16.5
 
-replace k8s.io/metrics => k8s.io/metrics v0.0.0-20181128195641-3954d62a524d
+replace k8s.io/metrics => k8s.io/metrics v0.0.0-20190819143841-305e1cef1ab1
 
-replace k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190221084156-01f179d85dbc
+replace k8s.io/apimachinery => k8s.io/apimachinery v0.16.5
 
-replace k8s.io/client-go => k8s.io/client-go v0.0.0-20190528110200-4f3abb12cae2
+replace k8s.io/client-go => k8s.io/client-go v0.16.5
 
-replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20190528110544-fa58353d80f3
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20190819143637-0dbe462fe92d
 
 replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
 
 replace github.com/sirupsen/logrus => github.com/jtnord/logrus v1.4.2-0.20190423161236-606ffcaf8f5d
 
-replace github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v21.1.0+incompatible
+replace github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v23.2.0+incompatible
 
-replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v10.14.0+incompatible
+replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.1+incompatible
+
+replace github.com/banzaicloud/bank-vaults => github.com/banzaicloud/bank-vaults v0.0.0-20191212164220-b327d7f2b681
+
+replace github.com/banzaicloud/bank-vaults/pkg/sdk => github.com/banzaicloud/bank-vaults/pkg/sdk v0.0.0-20191212164220-b327d7f2b681
+
+replace k8s.io/test-infra => github.com/abayer/test-infra v0.0.0-20200611142252-211a92405c22
```

Comment:
Should this fork of test-infra not live under the jenkins-x org? I don't think anything like this should be in personal repos. WDYT?

Reply:
Yeah, I’ll push that branch to the jenkins-x fork.

Reply:
Done - https://github.com/jenkins-x/test-infra/tree/for-k8s-1.16-deps